sqlstore/src/main/resources/db/migration/V8802__favourite_workflows.sql

```diff
@@ -1 +1 @@
-CREATE TABLE User_FavouriteWorkflows (userId BIGINT(20), favouriteWorkflow VARCHAR(20));
+CREATE TABLE User_FavouriteWorkflows (userId BIGINT(20), favouriteWorkflow VARCHAR(20), FOREIGN KEY (userId) REFERENCES User(UserId)) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
Sorry, I should have specified before - please name the constraint explicitly. This makes it easier to modify later. e.g. `CONSTRAINT fk_user_favouriteworkflow_user FOREIGN KEY (userId) ...`